app/controllers/events_controller.rb

```diff
@@ -1,24 +1,63 @@
 class EventsController < ApplicationController
   before_action :authenticate_user!
 
+
+  def index
+    @calendar = current_user.calendars.find params[:calendar_id]
```

Comment:
make a method `load_calendar` and move this logic to that method.

Reply:
Done